platform/core.multitabs/src/org/netbeans/core/multitabs/impl/TabDataRenderer.java

```diff
@@ -48,6 +53,12 @@
  */
 public class TabDataRenderer implements TableCellRenderer {
 
+    private static final Insets tabInsets = UIManager.getInsets("nb.multitabs.tabInsets"); // NOI18N
+    private static final int underlineHeight = UIManager.getInt("nb.multitabs.underlineHeight"); // NOI18N
+    private static final Color underlineColor = UIManager.getColor("nb.multitabs.underlineColor"); // NOI18N
+    private static final Color inactiveUnderlineColor = UIManager.getColor("nb.multitabs.inactiveUnderlineColor"); // NOI18N
+    private static final Color activeBackground = UIManager.getColor("nb.multitabs.activeBackground"); // NOI18N
```

Comment:
As before, when adding UIManager keys, I assume that you feel confident that they new keys are needed, and that you've figured out what the naming convention is for them...

Reply:
Yes, I've searched in the multitabs modules for other keys, which mostly use "nb.multitabs." as prefix. So I used the same prefix.

Reply:
Good.